src/main/java/BasicMAPF/Solvers/LargeNeighborhoodSearch/LargeNeighborhoodSearch_Solver.java

```diff
@@ -87,15 +89,21 @@ public LargeNeighborhoodSearch_Solver(I_SolutionCostFunction solutionCostFunctio
         this.sharedSources = Objects.requireNonNullElse(sharedSources, false);
         this.reactionFactor = Objects.requireNonNullElse(reactionFactor, 0.01);
         this.neighborhoodSize = Objects.requireNonNullElse(neighborhoodSize, 5);
+        this.TransientMAPFGoalCondition = Objects.requireNonNullElse(TransientMAPFGoalCondition, false);
```

Comment:
Take this line higher, and then when initializing PrP, use `this.TransientMAPFGoalCondition`. That way we enforce that they are consistent